README.md

```diff
@@ -54,6 +54,23 @@ async myMethod () {
 }
 ```
 
+#### Request body
+
+Instead of stringify the body of a request which the content type is `application/json`, you can just pass the javascript object as the body of the request and Request.JS will stringify it for you.
+
+```js
+import { post } from '@rails/request.js'
+
+...
+
+async myMethod () {
+  const response = await post('localhost:3000/my_endpoint', { body: { name: 'Request.JS' } })
```

Comment:
Doesn't this need `responseKind: 'json'` to trigger the stringify?

Reply:
What is required is the Content-Type to be `application/json` and this is the default content type in this scenario https://github.com/rails/request.js/blob/main/src/fetch_request.js#L73

Reply:
Ah! I didn't catch json as the default content type. 👍